lang/lang-es.js

```diff
@@ -0,0 +1,591 @@
+    uploadvideo: 'Subir vídeo',
+    uploadedsession: 'Su sesión ha finalizado. Puede cerrar la ventana. '
+    + ' o cierre esta ventana emergente para comenzar una nueva sesión.',
+    usermissing: 'Falta el alumno',
+    userList: 'Lista de usuarios',
+    upvote: 'Voto a favor',
+    instructorVideo: 'Video del docente',
+    validateurlmsg: 'URL no válida',
+    VCE2: 'No tiene permiso para grabar esta sesión',
+    VCE4: 'Faltan datos de registro',
+    VCE5: 'Incapaz de registrar datos.',
+    VCE6: 'Falta el módulo del curso.',
+    VideodbHeading: 'Panel del vídeo',
+    videoInput: 'Entrada de Vídeo',
+    videooff: 'Vídeo desactivado',
+    videoon: 'Vídeo activado',
+    videoquality: 'Calidad de video',
+    virtualclassnoteHeader: 'Añadir nota',
+    VotedSoFar: 'Votos recibidos hasta ahora',
+    votesuccess: 'Su voto ha sido enviado con éxito. El resultado se mostrará en breve.',
+    votesuccessPbt: 'Voto enviado con éxito, no se le permite ver el resultado.',
+    Vwllbshrshortly: 'El video se compartirá en breve',
+    viewall: 'Ver todo',
+    waitmsgconnect: 'Por favor, espere un momento. La aplicación está intentando conectarse.',
+    watstdrespo: 'Esperando la respuesta del alumno',
+    wbrtcMsgChrome: 'Puede hacer clic en el botón denegar para no compartir su micrófono y cámara con Congrea.'
+    + 'o haga clic en el botón Permitir para compartirlo.',
+    wbrtcMsgFireFox: 'Puedes hacer clic en  "Compartir dispositivos seleccionados"' 
+    + ' para compartir tu micrófono y cámara con otros usuarios.',
+    webcamerainfo: 'Si el video es visible, la webcam está funcionando correctamente.',
+    Whiteboard: 'Pizarra',
+    writemode: 'Modo de escritura',
+    youtubeshare: 'Compartir video de YouTube',
+    youTubeUrl: 'Introducir URL del vídeo de YouTube',
+    zoomIn: 'Más Zoom',
+    zoomOut: 'Menos Zoom',
+    noain: 'Sin entrada de audio',
+    noaout: 'Sin salida de audio',
+    novideo: 'No hay video',
+    enableAllAudio: 'Activar todo el audio',
+    disableAllAudio: 'Silenciar todo el audio',
+    colorSelector: 'Color',
+    strk: 'Tamaño del trazo',
+    font: 'Tamaño de la fuente',
+    size: 'Talla',
+    fullScreen: 'Pantalla completa',
+    exitFullScreen: 'Salir de pantalla completa',
+    recordingText: 'Rec',
+    recordingStopped: 'Grabación detenida',
+    recordingStarted: 'Grabación iniciada',
+    continueText: 'Clic aquí para continuar',
+    continue: 'Seguir',
+    moveSidebar: 'Mover barra lateral',
+    timeLimit: 'Límite de tiempo',
+    noOfQuestions: 'No de preguntas',
+    // chat files string
+    online: 'Usuarios en línea',
+    whos: 'Quién está en línea',
+    chatroom: 'Sala de chat',
+    chatroom_header: 'Chat común',
+    error: 'Ha ocurrido un error. Lo sentimos!',
+    wserror: 'Su navegador no es compatible con WebSocket',
+    sterror: 'Su navegador no es compatible con el estándar de almacenamiento web',
+    
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Trailing spaces not allowed.](https://app.codacy.com/gh/congrea/virtualclass/pullRequest?prid=5529949)